File: Solved Problems/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

while True:
    height = get_int("Give height\n")
    if height>0 and height<9:
        break
    else:
        print("Number not in range")

for i in range(height):
    for y in range(height-i-1):
        print(" ", end="")
    for x in range(i+1):
        print("#", end="")
    print("  ", end="")
    for z in range(i+1):
        print("#", end="")
    print("")

File: Solved Problems/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void printline(int nrSpaces, int nrHash)
{
    for(int i=0; i<nrHash+nrSpaces; i++)
    {
        if(i < nrSpaces)
        {
            printf(" ");
        }
        else
        {
            printf("#");
        }
    }
    printf("  ");
}

void printline2(int nrHash)
{
    for (int i=0; i<nrHash; i++)
    {
        printf("#");
    }
    printf("\n");
}

int main(void)
{
    int n = 0;
    while (true)
    {
        n = get_int("Input height: \n");
        if(n>0 && n<=8)
        {
            break;
        }

    }

    for( int i = 1; i <= n; i++)
    {
        printline(n-i,i);
        printline2(i);
    }
}